code/__defines/subsystem-defines.dm

```diff
@@ -90,3 +90,8 @@
 
 // -- SSicon_cache --
 #define LIGHT_FIXTURE_CACHE(icon,state,color) SSicon_cache.light_fixture_cache["[icon]_[state]_[color]"] || (SSicon_cache.light_fixture_cache["[icon]_[state]_[color]"] = SSicon_cache.generate_color_variant(icon,state,color))
+
+
+// -- SSmob_ai --
+#define START_THINKING(mob) if (!mob.thinking_enabled) { SSmob_ai.processing += mob; mob.on_think_enabled(); mob.thinking_enabled = TRUE; }
```

Comment:
I wonder if it'd be a smart idea to call these `MOB_START_THINKING` for added clarity?